aio/deploy/helm-chart/kubernetes-dashboard/templates/serviceaccount.yaml

```
@@ -0,0 +1,23 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.serviceAccount.create -}}
```

```suggestion
{{- if .Values.serviceAccount.create }}
```
Causes following error due to chomp basically moving `apiVersion` into the comment text:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: apiVersion not set
```
```

---

Here is what the result looks like:

```yaml
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.apiVersion: v1
kind: ServiceAccount
metadata:
  labels:
    app.kubernetes.io/name: kubernetes-dashboard
```

---

I think the actual problem is the `{{-` at the beginning. Since the top of the file has comments, we should remove all the whitespaces before `{{-`. What would work is:

```suggestion
{{ if .Values.serviceAccount.create -}}
```

---

Seems to work with the second `-` removed. Thank you very much!